src/gui/ExportProjectDialog.cpp

```diff
@@ -194,9 +194,13 @@ ProjectRenderer::ExportFileFormats convertIndexToExportFileFormat(int index)
 	case 0:
 		return ProjectRenderer::WaveFile;
 	case 1:
+#ifdef LMMS_HAVE_OGGVORBIS
 		return ProjectRenderer::OggFile;
 	case 2:
+#endif
+#ifdef LMMS_HAVE_MP3LAME
 		return ProjectRenderer::MP3File;
+#endif // If neither ogg nor mp3 are available, index 1 is OOB; fall through to default case
```

Comment:
This is hard to read and will be even harder to add new items to.  Let's instead parse "OGG, WAV, MP3" from the dropdown and make our assumptions there.